composio/local_tools/local_workspace/cmd_manager/actions/clone_github.py

```diff
@@ -17,11 +19,18 @@
 
 
 class GithubCloneRequest(BaseRequest):
-    github_token: str = Field(..., description="github token to clone the repository.")
+    workspace_id: str = Field(
+        ..., description="workspace id on which to clone the repo"
+    )
     repo_name: str = Field(
-        default="princeton-nlp/SWE-bench",
+        ...,
         description="github repository to clone. defaults to princeton-nlp/SWE-bench",
     )
+    commit_id: str = Field(
+        "",
```

Comment:
Why can't this be empty? I just want to clone to master and have no ideas what other commit's exist.

Reply:
changed this behavior